tests/utils/test_uri.py

```diff
@@ -99,10 +99,10 @@ def test_is_local_uri():
     assert is_local_uri("file://localhost:5000/mlruns")
     assert is_local_uri("file://127.0.0.1/mlruns")
     assert is_local_uri("file://127.0.0.1:5000/mlruns")
+    assert is_local_uri("file://myhostname/path/to/file")
```

Comment:
Is `file://myhostname` a local URI?

Reply:
Emm if moving scheme check up then yes, because the scheme is `file`

Reply:
But seems it's not a valid local URI..

Reply:
I think if they want to use a valid remote uri, they don't need to prepend with 'file'

Reply:
Let me read https://en.wikipedia.org/wiki/File_URI_scheme

Reply:
It's not

Reply:
According to https://en.wikipedia.org/wiki/File_URI_scheme, it is, right?

Reply:
Technically yes, but I think we should forbid it as it's super rare when hostname is not localhost for file://hostname/path/to/file.